docs/reference/inference/put-inference.asciidoc

```diff
@@ -257,11 +257,11 @@ It can be the ID of either a built-in model (for example, `.multilingual-e5-smal
 
 `num_allocations`:::
 (Required, integer)
-The number of model allocations to create. `num_allocations` must not exceed the number of available processors per node divided by the `num_threads`.
+The total number of allocations this model is assigned across machine learning nodes. Increasing this value generally increases the throughput.
```

Comment:
The old text is incorrect: it's not limited by the number of processors per node, because you can spin up many nodes.

The new text is copy/pasted from:
https://www.elastic.co/guide/en/elasticsearch/reference/current/start-trained-model-deployment.html#start-trained-model-deployment-query-params